src/InstagramClone.Api/Startup.cs

```diff
@@ -43,7 +44,24 @@ private static void Configure(WebApplication app)
 
             app.UseRouting();
 
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllerRoute(
+                    name: "default",
+                    pattern: "{controller=Home}/{action=Index}/{id?}");
+            });
```

Comment:
this call is duplicated below so delete this one and leave the one below.

Reply:
ok.